ci-operator/config/openshift/cluster-network-operator/openshift-cluster-network-operator-master.yaml

```diff
@@ -152,6 +152,12 @@ tests:
   steps:
     cluster_profile: azure4
     workflow: openshift-e2e-azure-ovn-dualstack
+- always_run: false
+  as: e2e-azure-manual-oidc
```

Comment:
change to `e2e-azure-ovn-manual-oidc` please. there is a test case that will flake if we don't
include the CNI in the job name.

Reply:
Does the flake happen specifically in tests ran against cluster network operator or should we rename all tests to include ovn across the board for other repositories?

Reply:
Based on how `make jobs` works I'm guessing we rename across the board.

Reply:
I think "most" jobs have this in their names at this point. It's a background project I've been working
on since 4.12. there are some that still need it, but hopefully not very many. I wouldn't worry about
other jobs with this PR though.